### PR TITLE
Ability to override config

### LINKED ISSUE #4: Add ability to override tslint and stylelint configs
Users may have their own custom styleguide, so we need an ability to override paths to linters configs via config.js file.

### PR DESCRIPTION
**Pull request by turbo-lover**
fix #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ module.exports = {
     projectRoot: __dirname,
     buildPath: path.resolve(__dirname, "public") // absolute path to your dist directory
     // generally its path to your public directory
+
+    //set relative path to project root folder to stlyleLint and tsLint config
```
Comment (turbo-lover) [inline README.md]: is this spelling is normal?

File: cli/commands/lint/ts.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
-module.exports = function (ffbt_root, workdir, argv) {
+module.exports = function (ffbt_root, workdir, argv, projectConfig, PROJECT_ROOT_PATH) {
     argv = argv || {};
     const fix = (typeof argv.fix === "undefined") ? false : Boolean(argv.fix);
     const force = (typeof argv.force === "undefined") ? false : Boolean(argv.force);
+    let configPath = path.resolve(ffbt_root, "./linters/tslint.json");
+
+    if(projectConfig.tsLintConfigPath) {
+        configPath = path.resolve(PROJECT_ROOT_PATH, projectConfig.tsLintConfigPath);
+        console.log("test", configPath);
```
Comment (arodik) [inline cli/commands/lint/ts.js]: logs

File: cli/commands/lint/ts.js [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
     tslintRunner = require("tslint/lib/runner");
 
-module.exports = function (ffbt_root, workdir, argv) {
+module.exports = function (ffbt_root, workdir, argv, projectConfig, PROJECT_ROOT_PATH) {
     argv = argv || {};
     const fix = (typeof argv.fix === "undefined") ? false : Boolean(argv.fix);
     const force = (typeof argv.force === "undefined") ? false : Boolean(argv.force);
+    let configPath = path.resolve(ffbt_root, "./linters/tslint.json");
+
+    if(projectConfig.tsLintConfigPath) {
```
Comment (arodik) [inline cli/commands/lint/ts.js]: missed whitespace

File: cli/commands/lint/style.js [hunk 4]
```diff
@@ -2,9 +2,15 @@ const path = require("path"),
...
 
 let noErrorExitCode;
-module.exports = function (ffbt_root, workdir, argv) {
+module.exports = function (ffbt_root, workdir, argv, projectConfig, PROJECT_ROOT_PATH) {
     noErrorExitCode = argv && argv.force;
-    const configPath = path.resolve(ffbt_root, "./linters/stylelint.config.js");
+
+    let configPath = path.resolve(ffbt_root, "./linters/stylelint.config.js");
+
+    if(projectConfig.styleLintConfigPath) {
```
Comment (arodik) [inline cli/commands/lint/style.js]: missed whitespace

File: README.md (hunk 1 above)
Comment (arodik) [reply README.md]: Would be better if we say "You can specify the path to your custom lint config if the default config doesn't fit your needs"

**Review by arodik — APPROVED** [review]

**Pull request merged**